### PR TITLE
Multi select bar - style and icon fixes

### PR DESCRIPTION
**Pull request by fritzSF**

### CODE REVIEW

File: client/app/styles/less/sf-icons.less [hunk 1]
```diff
@@ -523,6 +523,67 @@ a:hover .big-icon-info {
...
...4wOTcgMjAuMzIsMjQuMDk3IDIwLjcwNiwyMy43MUwyMC43MTIsMjMuNzA1QzIxLjA5OCwyMy4zMTggMjEuMDk4LDIyLjY5MiAyMC43MTIsMjIuMzA1IiBzdHlsZT0iZmlsbDpyZ2IoMCwxMjcsMTc5KTtmaWxsLXJ1bGU6bm9uemVybzsiLz4KICAgICAgICA8Zz4KICAgICAgICAgICAgPGcgb3BhY2l0eT0iMC4zOTk5OTQiPgogICAgICAgICAgICAgICAgPHBhdGggZD0iTTE1LDE0TDE1LDlMMTMsOUwxMywxMkwxNSwxNFoiIHN0eWxlPSJmaWxsOnJnYigwLDEyNywxNzkpO2ZpbGwtcnVsZTpub256ZXJvOyIvPgogICAgICAgICAgICA8L2c+CiAgICAgICAgPC9nPgogICAgICAgIDxnPgogICAgICAgICAgICA8ZyBvcGFjaXR5PSIwLjM5OTk5NCI+CiAgICAgICAgICAgICAgICA8cGF0aCBkPSJNMTMsMTcuMzQ0TDEzLDIwTDE1LDIwTDE1LDE5LjM0NEwxMywxNy4zNDRaIiBzdHlsZT0iZmlsbDpyZ2IoMCwxMjcsMTc5KTtmaWxsLXJ1bGU6bm9uemVybzsiLz4KICAgICAgICAgICAgPC9nPgogICAgICAgIDwvZz4KICAgICAgICA8Zz4KICAgICAgICAgICAgPGcgb3BhY2l0eT0iMC4zOTk5OTQiPgogICAgICAgICAgICAgICAgPHBhdGggZD0iTTE5LDE4TDE5LDlMMTcsOUwxNywxNkwxOSwxOFoiIHN0eWxlPSJmaWxsOnJnYigwLDEyNywxNzkpO2ZpbGwtcnVsZTpub256ZXJvOyIvPgogICAgICAgICAgICA8L2c+CiAgICAgICAgPC9nPgogICAgICAgIDxnPgogICAgICAgICAgICA8ZyBvcGFjaXR5PSIwLjM5OTk5NCI+CiAgICAgICAgICAgICAgICA8cGF0aCBkPSJNMTAuMDE2LDlMMTEsOS45ODRMMTEsOUwxMC4wMTYsOVoiIHN0eWxlPSJmaWxsOnJnYigwLDEyNywxNzkpO2ZpbGwtcnVsZTpub256ZXJvOyIvPgogICAgICAgICAgICA8L2c+CiAgICAgICAgPC9nPgogICAgPC9nPgo8L3N2Zz4K);
```
Comment (plamut) [inline client/app/styles/less/sf-icons.less]: Just curious - why are all these icons defined inline? Is this better than using a sprite?

**Comment by plamut** [comment]
Looks good, though it seems that a few selectors in e2e tests will have to be updated due to the changed CSS classes of some elements.

File: client/app/styles/less/sf-icons.less (hunk 1 above)
Comment (fritzSF) [inline client/app/styles/less/sf-icons.less]: Those are vector icons so sprite isn't really an option.

Comment (plamut) [inline client/app/styles/less/sf-icons.less]: Aha, thanks for explaining!

**Pull request merged**